src/main/java/exercise39/SortingRecords.java

```diff
@@ -60,8 +60,11 @@ public String recordSorter(List<Map<String, String>> records) {
     public String recordFilter(String searchTerm, List<Map<String, String>> records) {
 
         String filteredRecords = "";
+        //removeIf?
```

Comment:
Looks good, just for the practice, can you think of how to rewrite it using the `ArrayList.removeIf` method instead of the loop.  It'll be good practice for lambdas and predicates.